app/views/edit-health.html

```diff
@@ -0,0 +1,138 @@
+                                <option value="ops">Ops</option>
+                                <option value="userResearch">User Research</option>
+                                <option value="security">Security</option>
+                                <option value="delivery">Delivery</option>
+                                <option value="data">Data</option>
+                            </select>
+                        </div>
+
+                        <!-- status -->
+                        <div class="form-group">
+                            <label class="form-label-bold" for="status">
+                                Healthcheck Status
+                                <span class="form-hint">
+                                    Health of the project
+                                </span>
+                            </label>
+
+                            <select class="form-control" name="health[status]" id="status">
+                                <option value="good">Good</option>
+                                <option value="fair">Fair</option>
+                                <option value="risk">Risk</option>
+                                <option value="unknown">Unknown</option>
+                            </select>
+                        </div>
+
+                        <!-- comment -->
+                        <div class="form-group">
+                            <label class="form-label-bold" for="comment">
+                                Comment
+                                <span class="form-hint">
+                                    Reason for the health status change
+                                </span>
+                            </label>
+                            <textarea class="form-control" id="comment" name="health[comment]" maxlength="500">{{ project.health.comment }}</textarea>
+                        </div>
+
+                        <!-- link title-->
+                        <div class="form-group">
+                            <label class="form-label-bold" for="name">
+                                Healthcheck Link
+                                <span class="form-hint">
+                                    Title of the link
+                                </span>
+                            </label>
+                            <input type="text" name="health[link][name]" id="name" placeholder="Example Title"></input>
+                                <span class="form-hint">
+                                    Web address of the link
+                                </span>
+                            <input type="url" name="health[link][url]" placeholder="http://example.com/"></input>
+                        </div>
+
+                        <div class="form-group">
+                            <input type="submit" class="button" value="Submit">
+                        </div>
+                    </form>
+
+                    {% if project["healthHistory"] %}
```

Comment:
this is the old healthHistory, we can remove it untill we re-write it for the new version